scripts/build.py

```diff
@@ -462,18 +602,21 @@ def get_file_hashes(directory_path):
 
     Returns:
         dict(str, str). Dictionary with keys specifying file paths and values
-        specifying file hashes.
+            specifying file hashes.
     """
     file_hashes = dict()
 
     print('Computing hashes for files in %s'
           % os.path.join(os.getcwd(), directory_path))
 
-    for root, dirs, files in os.walk(os.path.join(os.getcwd(), directory_path)):
-        for directory in dirs:
+    for root, dirnames, filenames in os.walk(
+            os.path.join(os.getcwd(), directory_path)):
+        for directory in dirnames:
             print('Computing hashes for files in %s'
```

Comment:
I know this was there before, but just thinking, does this actually print out things in the right order? If I'm reading this right it feels like it prints out the strings for all the dirnames before actually processing any of the files, but it's late and I might have misunderstood.

Same question elsewhere, any time we os.walk().

Reply:
The actual file processing is done later with multithreading, which has its own message printed in `minify_func`.

The messages here is signaling the `os.walk` is going through `directory_path` and `os.walk` is iterating through each sub-folder in `directory_path` to "compute hashes for files"

Reply:
Ah ok, thanks! I forgot that the messages get printed before we actually do the thing (i.e. with the implication of "We're going to do this ..." rather than "We're doing this right now ..."). Thanks!)